Module/Rule/Convert/Data.McAfee.ps1

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+    Instructions:  Use this file to add/update/delete regsitry expressions that are used accross 
+    multiple technologies files that are considered commonly used.  Ensure expressions are listed
+    from MOST Restrive to LEAST Restrictive, similar to exception handling.  Also, ensure only
+    UNIQUE Keys are used in each hashtable to prevent errors and conflicts.
+#>
+
+$global:SingleLineRegistryValueName += [ordered]@{
```

Comment:
Do these need to be [global variables](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#avoid-global-variables) or with they work in the script: scope?